http/http_client.hpp

```diff
@@ -172,23 +177,29 @@ class HttpClient : public std::enable_shared_from_this<HttpClient>
         req.body() = data;
         req.prepare_payload();
 
-        auto respHandler =
-            [self(shared_from_this())](const boost::beast::error_code ec,
-                                       const std::size_t& bytesTransferred) {
-                if (ec)
-                {
-                    BMCWEB_LOG_ERROR << "sendMessage() failed: " << ec.message()
-                                     << " to subId: " << self->subId;
-                    self->state = ConnState::sendFailed;
-                    self->handleConnState();
-                    return;
-                }
+        auto respHandler = [self(shared_from_this())](
+                               const boost::beast::error_code ec,
+                               const std::size_t& bytesTransferred) {
+            if (ec)
+            {
+                std::string_view eventStr = self->req.body();
+                nlohmann::json event;
+                auto eventData = event.parse(eventStr);
```

Comment:
If you are going to do eventData"Id" below shouldn't you check for .is_discarded() https://github.com/ibm-openbmc/bmcweb/blob/8044381620b228f99b707ea2b444c7675e714f26/include/login_routes.hpp#L42 ?

Reply:
Its not a user input. This is a json which is built within the BMC. A event record data. So it can not go bad.